Tests/UnitTests/Core/Utilities/BinUtilityTests.cpp

```diff
@@ -42,6 +42,11 @@ BOOST_AUTO_TEST_CASE(BinUtility_equidistant_binning) {
   BOOST_CHECK_EQUAL(xyUtil_eq.dimensions(), 2u);
   BOOST_CHECK_EQUAL(xyzUtil_eq.dimensions(), 3u);
 
+  // check equality operator
+  BinUtility xUtil_eq_copy(10, 0., 10., open, binX);
+  BOOST_CHECK(xUtil_eq_copy == xUtil_eq);
+  BOOST_CHECK(not(yUtil_eq == xUtil_eq));
```

Comment:
You don't need to change this, but there are `BOOST_CHECK_EQUAL(A, B)` and `BOOST_CHECK_NE(A, B)` which give nicer errors if they fail.

Reply:
I will change.